exporter/SynthesisFusionAddin/src/Parser/SynthesisParser/Parser.py

```diff
@@ -178,9 +182,9 @@ def export(self) -> None:
             folder_id = project.rootFolder.id
             file_name = f"{self.exporterOptions.fileLocation}.mira"
             if upload_mirabuf(project_id, folder_id, file_name, assembly_out.SerializeToString()) is None:
-                gm.ui.messageBox("FAILED TO UPLOAD FILE TO APS", "ERROR")  # add throw later
-        # Download Mirabuf File
+                raise RuntimeError("Could not upload to APS")
         else:
+            assert self.exporterOptions.exportLocation == ExportLocation.DOWNLOAD
```

Comment:
I forgot that enums could be undefined 🤦‍♀️

Reply:
I'm more so just checking to make sure we don't add different export locations in the future. Sort of a "this was the entire behaviour of `ExportLocation` when I wrote this code so it may need to be updated" type of thing. But yea I suppose it will also catch if it is `None` as well.